packages/markdown/src/vaadin-markdown.js

```diff
@@ -43,10 +44,33 @@ class Markdown extends ElementMixin(ThemableMixin(PolylitMixin(LitElement))) {
     `;
   }
 
+  static get properties() {
+    return {
+      /**
+       * The Markdown content.
+       *
+       * @type {string}
+       */
+      markdown: {
+        type: String,
+      },
+    };
+  }
+
   /** @protected */
   render() {
     return html`<slot></slot>`;
   }
+
+  /**
+   * @protected
+   * @override
+   */
+  updated(props) {
+    super.updated(props);
+
+    renderMarkdownToElement(this, this.markdown);
```

Comment:
I would add a check for `if (props.has('markdown'))` just in case. There are no other props in the component but technically `theme` attribute from `ThemePropertyMixin` triggers an internal property which causes update.

Reply:
Done